modules/oc-mirror-updating-use-cases.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-image-set-examples-add-images_{context}"]
+= Use cases
+
+This section covers the use cases for updating the mirror registry from disk to mirror.
```

Comment:
Hi @snarayan-redhat 
I think we' re saying disk to mirror because these apply these use cases probably apply to prunin... in this case miror to mirror applies as well

Reply:
yes, that's right. We can address the mirror to mirror scenario too in upcoming PRs.